aws/datalake/README.md

```diff
@@ -0,0 +1,63 @@
+# Data Lake Solution on Amazon EC2
```

Comment:
So this Readme will be used by marketing and also for people who clone/view the repository, am I right?

Reply:
Yep, exactly.